tomviz/OperatorPython.cxx

```diff
@@ -118,6 +120,11 @@ void OperatorPython::registerCustomWidget(const QString& key, bool needsData,
                          QPair<bool, CustomWidgetFunction>(needsData, func));
 }
 
+void OperatorPython::dataUpdated()
+{
+  dataSource()->dataChanged();
+}
+
```

Comment:
Is this used?

Reply:
Nope. False start, will remove.

Reply:
Shall we remove this?

Reply:
Hmm, thought I removed this. I will really do that this time :-)